demo

```diff
@@ -182,214 +202,254 @@ exitfunc() {
-    sleep 5
-done
-
-
-echo "Creating a new user..."
-UPLOAD_ARTIFACT=1
-PRINT_LOGIN_INFO=0
-PRINT_USER_EXISTS=1
-if [[ $ENTERPRISE -eq 1 ]]; then
-    TENANT_ID=$(docker exec \
-            ${DOCKER_COMPOSE_PROJECT_NAME}_mender-tenantadm_1 \
-            /usr/bin/tenantadm create-org \
-            --name=DemoOrganization \
-            --username=${USER} \
-            --password=${PASSWORD})
-    retval=$?
-    EXISTS_ERROR=1
-    if [[ $retval -eq 0 ]]; then
-        TENANT_ID=$(echo "$TENANT_ID" | tr -d '\r')
-    else
-        TENANT_ID=
-    fi
-else
-    docker exec \
-            ${DOCKER_COMPOSE_PROJECT_NAME}_mender-useradm_1 \
-            /usr/bin/useradm create-user \
-            --username=${USER} \
-            --password=${PASSWORD} \
-            > /dev/null
-    retval=$?
-    EXISTS_ERROR=5
-fi
-if [[ $retval -eq 0 ]]; then
-    PRINT_LOGIN_INFO=1
-elif [[ $retval -eq $EXISTS_ERROR ]]; then
-    # If the user exists, skip uploading the Artifact
-    UPLOAD_ARTIFACT=0
-    PRINT_USER_EXISTS=1
-else
-    echo "docker exec error: " $retval
-    exit $retval
-fi
-
-if [[ $ENTERPRISE -eq 1 ]] && [[ $CLIENT -eq 1 ]]; then
-    if [[ $(docker ps -q -f name=${DOCKER_COMPOSE_PROJECT_NAME}_mender-client_1 | wc -l) -gt 0 ]]; then
-        # If already launched, we don't need to do anything.
-        :
-    elif [[ -n "$TENANT_ID" ]]; then
-        TENANT_TOKEN=$(docker exec \
-                ${DOCKER_COMPOSE_PROJECT_NAME}_mender-tenantadm_1 \
-                /usr/bin/tenantadm get-tenant \
-                --id $TENANT_ID \
-                | jq -r .tenant_token)
-        # Now that we have the tenant token we can enable the client.
-        EXTRA_FILES="$EXTRA_FILES $CLIENT_FILES"
-        TENANT_TOKEN=$TENANT_TOKEN docker-compose \
-                -f docker-compose.yml \
-                -f docker-compose.storage.minio.yml \
-                -f docker-compose.demo.yml \
-                -p ${DOCKER_COMPOSE_PROJECT_NAME} \
-                $EXTRA_FILES \
-                "$@" -d mender-client
-    else
-        echo "WARNING: Ignoring request to launch the Mender client."
-        echo "In Enterprise mode, the client can only be launched the first time the server"
-        echo "is started, when the first user is created. If you wish to start from scratch,"
-        echo "replace \`up\` with \`down -v\` first to reset, then rerun."
+    local retval=$?
+    if [[ $retval -ne 0 ]]; then
+        echo "Failed to start docker compose"
+        exit $retval
     fi
-fi
 
-if [[ $UPLOAD_ARTIFACT -eq 1 ]]; then
-
-    RETRIES=0
-    until [[ "$JWT" != "" ]]; do
-        JWT=$(curl --silent -k -X POST -u ${USER}:${PASSWORD}\
-            --fail\
-            --connect-timeout 5\
-            $MENDER_SERVER_URI/api/management/v1/useradm/auth/login)
+    # Block until the useradm service returns an HTTP 4xx response
+    local RETRIES=0
+    while :
+    do
+        curl --silent -k -X POST -u ${USER}:${PASSWORD} \
+             --fail\
```

Comment:
Nitpick:
5 spaces

Reply:
This is completely inconsistent throughout the whole script, even in the original. I'll make it 8 spaces, double of the base indentation, as long as it's the same concatenated line.